<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FaceWidget</class>
 <widget class="QWidget" name="FaceWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1731</width>
    <height>1351</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>6</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Orientation::Vertical</enum>
     </property>
     <widget class="EyesWidget" name="widgetEyes" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
     </widget>
     <widget class="RealtimeValuesWidget" name="widgetRealtimeValues" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
     </widget>
     <widget class="LogWidget" name="logScroll">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widgetConnect" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="TryToggle" name="tryToggleConnect" native="true">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>10</width>
            <height>20</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButtonNewColor">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>20</height>
           </size>
          </property>
          <property name="text">
           <string>Color</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="PanicButton" name="pushButtonPanic">
          <property name="text">
           <string>Panic</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>LogWidget</class>
   <extends>QPlainTextEdit</extends>
   <header location="global">node/LogWidget.hpp</header>
  </customwidget>
  <customwidget>
   <class>TryToggle</class>
   <extends>QWidget</extends>
   <header location="global">components/TryToggle.hpp</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>EyesWidget</class>
   <extends>QWidget</extends>
   <header location="global">expression/EyesWidget.hpp</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>RealtimeValuesWidget</class>
   <extends>QWidget</extends>
   <header location="global">agent/RealtimeValuesWidget.hpp</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>PanicButton</class>
   <extends>QPushButton</extends>
   <header location="global">node/PanicButton.hpp</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>splitter</sender>
   <signal>splitterMoved(int,int)</signal>
   <receiver>FaceWidget</receiver>
   <slot>onSplitterMoved(int,int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>865</x>
     <y>658</y>
    </hint>
    <hint type="destinationlabel">
     <x>865</x>
     <y>675</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButtonNewColor</sender>
   <signal>clicked()</signal>
   <receiver>FaceWidget</receiver>
   <slot>onColorClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>865</x>
     <y>1328</y>
    </hint>
    <hint type="destinationlabel">
     <x>865</x>
     <y>675</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButtonPanic</sender>
   <signal>toggled(bool)</signal>
   <receiver>FaceWidget</receiver>
   <slot>onPanicToggled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1443</x>
     <y>1337</y>
    </hint>
    <hint type="destinationlabel">
     <x>865</x>
     <y>675</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>onSplitterMoved(int,int)</slot>
  <slot>onPanicToggled(bool)</slot>
  <slot>onColorClicked()</slot>
 </slots>
</ui>
